docs/data-sources/connect-to-a-source.mdx

```diff
@@ -49,6 +49,14 @@ it requires:
 
 <Tabs groupId="source-preference" className="api-tabs">
 
+<TabItem value="BigQuery" label="BigQuery">
+
+| ENV        | Example                                                                                                                                                                                               | Description                                       |
+| ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------- |
+| `JDBC_URL` | `jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443;ProjectId=project-id;DefaultDataset=dataset;OAuthType=0;OAuthServiceAcctEmail=service-account-email;OAuthPvtKey=/etc/connector/key.json;` | The JDBC URL to connect to the BigQuery database. |
```

Comment:
Would it be helpful to add a note below this where we educate the user a bit on the last param? It could be something minimal like, `See [this guide](ref) for instructions on saving your service account key file.`

Reply:
👍